package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "file-changer.js",
+  "scripts": {
+    "lint": "eslint '**/*.js'",
+    "test-watch": "jest --watchAll",
+    "test": "jest --verbose --coverage",
+    "jsdoc": "jsdoc -c ./docs/config/jsdoc.config.json"
+  },
+  "author": "Morgan T Shaw",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
You need to install `socket.io` and `socket.io-client` to your app.  they should be in the dependencies.